File: POOUber/php/card.php
<?php
require_once('payment.php');

class Card extends Payment{
    public $cardNumber;
    public $cvv;
    public $date;

    public function __construct($id, $cardNumber, $cvv, $date){
        parent::__construct($id);
        $this->cardNumber = $cardNumber;
        $this->cvv = $cvv;
        $this->date = $date;
    }
}

File: POOUber/java/card.java
class Card extends Payment{
    String cardNumber;
    String cvv;
    String date;
    public Card(int id, String cardNumber, String cvv, String date){
        super(id);
        this.cardNumber = cardNumber;
        this.cvv = cvv;
        this.date = date;
    }
}